### PR TITLE
Github actions

### PR DESCRIPTION
**Pull request by jonathanfrawley**
Add GitHub action for integration test for test repo.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Install dependencies
+      if: steps.cache-pip.outputs.cache-hit != 'true'
+      run: |
+        pip install Cython
+        pip install numba==0.52.0
+        pip install numpy==1.20.0
```
Comment (rhayes777) [inline .github/workflows/main.yml]: numpy must already be in the requirements?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+name: Autolens Workspace Test Integration Test
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        #python-version: [3.6, 3.7, 3.8, 3.9]
```
Comment (rhayes777) [inline .github/workflows/main.yml]: I think we are backwards compatible for 3.6 and 3.7 ?

**Review by rhayes777 — APPROVED** [review]

File: .github/workflows/main.yml (hunk 2 above)
Comment (jonathanfrawley) [reply .github/workflows/main.yml]: Cool, good to know. I'll enable.

File: .github/workflows/main.yml (hunk 1 above)
Comment (jonathanfrawley) [reply .github/workflows/main.yml]: Yes, sorry I had that in to test a fix to the numpy issue. Removed now.

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -14,28 +14,28 @@ jobs:
     - uses: actions/checkout@v2
     - name: Checkout PyAutoFit
       uses: actions/checkout@v2
-      ref: 'develop'
       with:
         repository: rhayes777/PyAutoFit
         path: PyAutoFit
+        ref: 'develop'
```
Comment (Jammy2211) [inline .github/workflows/main.yml]: develop branches have now been removed.

**Review by Jammy2211 — APPROVED** [review]

**Pull request merged**